abi/codecForSimpleValues_address.go

```diff
@@ -0,0 +1,48 @@
+package abi
+
+import (
+	"fmt"
+	"io"
+)
+
+func (c *codec) encodeNestedAddress(writer io.Writer, value AddressValue) error {
+	return c.encodeTopLevelAddress(writer, value)
```

Comment:
If you want to be consistent with codec from struct or from js-core you could make the check+write here and encodeTopLevel use the nested implementation